dynamic/org.ops4j.pax.logging.cfg

```diff
@@ -57,7 +57,9 @@ log4j2.appender.rolling.layout.type = PatternLayout
 log4j2.appender.rolling.layout.pattern = ${log4j2.pattern}
 log4j2.appender.rolling.policies.type = Policies
 log4j2.appender.rolling.policies.size.type = SizeBasedTriggeringPolicy
-log4j2.appender.rolling.policies.size.size = 2MB
+log4j2.appender.rolling.policies.size.size = 250KB
+log4j2.appender.rolling.strategy.type = DefaultRolloverStrategy
+log4j2.appender.rolling.strategy.max = 2
```

Comment:
Enable logging to file-system, but only need minimal size so that when the agent crashes we can fetch the log-files easily from disk. Use two files, because if one would get rolled-over, we'd have an empty file.
The files will be cleaned automatically when the container stops.